off_chain_data/README.md

```diff
@@ -25,7 +25,7 @@ Use the following command to start the sample network:
 This command will deploy an instance of the Fabric test network. The network
 consists of an ordering service, two peer organizations with one peers each, and
 a CA for each org. The command also creates a channel named `mychannel`. The
-marbles chaincode will be installed on both peers and deployed to the channel.
+assets chaincode will be installed on both peers and deployed to the channel.
```

Comment:
Wrong chaincode name, will fix.

Reply:
Should we say the full name of the chaincode, "asset transfer basic"

Reply:
Yup!

Reply:
done.